R/examples.R

```diff
@@ -0,0 +1,54 @@
+AddOne <- function(jaspResults, dataset, options) {
```

Comment:
I thought the style guide says camelCase: 
this should be `addOne`

Reply:
Done!
78eb154